region/brinstar/pink/Pink Brinstar Wave Gate Room.json

```diff
@@ -179,6 +184,59 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Entry",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "clearsObstacles": ["R-Mode"]
```

Comment:
```suggestion
      "clearsObstacles": ["R-Mode"],
      "devNote": [
        "FIXME: It is possible to avoid a forced Small Sidehopper hit by buffering an aim down as you release X-Ray.",
        "Evading further hits from that position would require more logic."
      ]
```